.github/ci_settings.py

```diff
@@ -22,4 +22,6 @@
 PG14_LATEST = "14.6"
 PG14_ABI_MIN = "14.0"
 
-PG_LATEST = [PG12_LATEST, PG13_LATEST, PG14_LATEST]
+PG15_LATEST = "15.1"
```

Comment:
Maybe it would be good to introduce the `PG15_EARLIEST` setting with this commit as well. Or are there reasons that prevent this?